proposals/lip-0049.md

```diff
@@ -179,19 +175,23 @@ crossChainMessageSchema = {
             "fieldNumber": 1
```

Comment:
Optional: For the schema, I would find it nice to follow the same order of properties as for transactions as much as possible, i.e., have `nonce` and `fee` directly after the `module` and `command`.

Reply:
sounds good, updated